x-pack/plugins/stack_connectors/public/connector_types/stack/email/email_params.tsx

```diff
@@ -114,6 +117,7 @@ export const EmailParamsFields = ({
             if (!to) {
               editAction('to', [], index);
             }
+            onBlur?.('to');
```

Comment:
[nit] the null check is not necessary since `onBlur` defaults to `noop`.

```suggestion
            onBlur('to');
```

Another way to fix it is to keep here the null check but not have a default callback, but I'll leave that to you.